packages/backend/src/Tabulators/Star.test.ts

```diff
@@ -115,11 +115,13 @@ describe("STAR Tests", () => {
     test("Test valid/invalid/under/bullet vote counts", () => {
         const candidates = ['Allison', 'Bill', 'Carmen']
         const votes = [
+            [5, 5, 5],
```

Comment:
oh, another edge case to consider is ``[0, null, null]`` . That should also be abstain since null counts as zero in STAR. I think adding back some ``?? 0`` expressions in the equality checks should do the trick

Reply:
ah ok. thanks!